### ISSUE TITLE
Dev Menu links in Chrome and Chromium MS Edge not resolving

### ISSUE THREAD

**Issue by MarkLopresto** [TheWebTech/HubSpot-Developer-Extension#235]
**Describe the bug**
Links in Developer menu are no longer resolving properly. When inspected, the link seems to be doubling protocol and domain name.

**To Reproduce**
1. Go to Portal
2. Click on Developer dropdown
3. Hover or click on any menu item

**Expected behavior**
Link to resolve to page

**Screenshots**
<img width="496" alt="Screenshot 2020-04-24 19 42 40" src="https://user-images.githubusercontent.com/6628656/80265525-d6b18b00-8665-11ea-8a3d-d750ffe1704a.png">

**Desktop (please complete the following information):**
 - OS: macOS 10.15.4
 - Browser: Microsoft Edge ( chromium )
 - Version: 81.0.416.64
 - Browser: Chrome
 - Version: 81.0.4044.122

**Additional context**
The Content Settings link works, but resolves with warning about problem loading account name setting.

**Comment by MarkLopresto** [comment]
Update: I discovered this issue only occurs if changing your Default Home Page, in Basic info, to anything other than Dashboards.

**Comment by TheWebTech** [comment]
Yep this is confirmed, had a second report of this exact same situation
![Screen Shot 2020-06-05 at 10 28 03 AM](https://user-images.githubusercontent.com/5492388/83894791-67748f80-a717-11ea-94cd-96138af0d76e.png)

**Comment by TheWebTech** [comment]
Related code:
https://github.com/TheWebTech/HubSpot-Developer-Extension/blob/7e6588f8f3ed595a122c784af4e1243c858ede5b/design-manager.js#L262-L314
